### ISSUE TITLE
Create new calendar if a Chapter loses access

### ISSUE THREAD

**Issue by ojeytonwilliams** [freeCodeCamp/chapter#1896]
### Discuss your ideas or share your views: 

Right now we can create Google calendar events if a Chapter event does not have one.  However, it's not possible to create new calendars at all.

In the event that the Chapter app is unable to modify the a calendar (presumably there's a request it can do to check this), it would be nice to be able to create a new one.

P.S. this is very much not MVP worthy, but I want to give people a chance to see this before I leave this to the roadmap.

**Comment by gikf** [comment]
I was going to add creating calendar in chapter dashboard, in similar manner to creating calendar events from event dashboard. It got somewhat delayed due to my not being sure how to approach the _checking if Google calendar is authorized and if integration is working_.

**Comment by ojeytonwilliams** [comment]
Yeah, it's not completely obvious and it's not a *huge* priority.  I only ran into it because I manually updated the staging db to fix an issue, so I don't see it happening much.

Having said all that, my vague thinking was that we could show the button in one of two situtations

- When we catch errors while trying to create an event
- After making a deliberate API call to check if we have access 

I don't know which is better, but I slightly prefer the first approach as long as we're sure that the error is caused by a lack of permission.
